src/routes/index.js

```diff
@@ -5,6 +5,7 @@ const allocationsRouter = require("./allocations")
 const vebalsRouter = require("./vebals")
 const nftvolsRouter = require("./nft_vols")
 const rewardsInfoRouter = require("./rewards_info")
+const rewardsSummaryRouter = require("./rewards_summary")
```

Comment:
We should add a route for querying passive rewards.

Reply:
rewardsSummary has both active & passive.   do we need a separate route just for passive ?